app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
I guess it is better to inherit from Exception here.
For more info you read via link: [https://peps.python.org/pep-0352/](url)